html2/q2.js

```diff
@@ -0,0 +1,156 @@
+const movies=[
+    
+  {
+    "MovieName": "The Great Adventure",
+    "ActorName": "John Smith",
+    "ReleaseDate": "2023-01-15"
+  },
+  {
+    "MovieName": "Mystery in the Woods",
+    "ActorName": "Emily Johnson",
+    "ReleaseDate": "2022-09-28"
+  },
+  {
+    "MovieName": "Love and Destiny",
+    "ActorName": "Michael Brown",
+    "ReleaseDate": "2023-05-02"
+  },
+  {
+    "MovieName": "City of Shadows",
+    "ActorName": "Sophia Williams",
+    "ReleaseDate": "2023-03-12"
+  },
+  {
+    "MovieName": "The Last Stand",
+    "ActorName": "William Davis",
+    "ReleaseDate": "2022-11-07"
+  },
+  {
+    "MovieName": "Echoes of Time",
+    "ActorName": "Olivia Wilson",
+    "ReleaseDate": "2022-12-19"
+  }
+
+];
+
+//1. List the movie name along with the actor name of those movies released in the year 2022
+console.log(movies
+    .filter(x=> new Date(x.ReleaseDate).getFullYear() === 2022)
+    .map(x=>`Movie : ${x.MovieName} , Actor : ${x.ActorName}`));
```

Comment:
this would return a string[], return obj[] instead

Reply:
Updated with this:
console.log(movies
    .filter(x=> new Date(x.ReleaseDate).getFullYear() === 2022)
    .map(x=>({MovieName:x.MovieName ,ActorName:x.ActorName})));